### ISSUE TITLE
No Connected line in condump

### ISSUE THREAD

**Issue by bitdivision** [light2yellow/csgo-csmd#1]
My condump file doesn't seem to have a 'Connected' line and is erroring out.

Any ideas if there's a way to increase the number of lines in the dump?

**Comment by bitdivision** [comment]
Only appears to be an issue on servers with an awful lot of files. cl_downloadfilter would probably help.

Thank you for making this! :+1:

**Comment by light2yellow** [comment]
Yes, indeed, now I recalled that `condump` can only hold 241 lines :)
I guess I can make an option to use `con_log`.

**Comment by light2yellow** [comment]
Could you provide such server (`ip:port`) with a lot of files, for testing?

**Comment by bitdivision** [comment]
Sorry for the delay: 95.172.92.196:27015

That is ze_gris_v1_13 at the moment.

**Comment by TheRealSkoopy** [comment]
I have the same problem. Most of the community servers run a lot of custom assets. condump isn't long enough. Script runs into error at line 205.

**Comment by bitdivision** [comment]
As a temp workaround, you can run `con_logfile consolelog.log` which will make a file called consolelog.log that always has your console history in it. Then change the script so that `CONDUMP = CSGO_HOME_DIR.joinpath("consolelog.log")`

Obviously you'll need to revert those changes if you want to use condump again.

**Comment by light2yellow** [comment]
> Then change the script so that CONDUMP = CSGO_HOME_DIR.joinpath("consolelog.log")

A new version has since been released that doesn't allow this, but in general, @bitdivision is correct. With the latest version one would have to check the highest `condumpXXX.txt` in the CS:GO homedir (i.e. find out that `XXX` number) and issue `log_file "condumpXXX.txt"`, where `XXX` is that number plus one (it should match the regex: `(condump)(\\d)(\\d)(\\d)(\\.)(txt)`). After the script has been used issue `log_file ""` so that it doesn't continue writing.

**Comment by light2yellow** [comment]
I actually meant `con_logfile`, not `log_file`.
